src/main/java/be/syntra/devshop/DevshopBack/controller/TstRestController.java

```diff
@@ -0,0 +1,16 @@
+package be.syntra.devshop.DevshopBack.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/devshop")
+public class TstRestController {
+
+    @GetMapping(value = "/test", produces = "application/JSON")
+    public ResponseEntity<String> displayTstString() {
+        return ResponseEntity.ok().body("DevShop");
```

Comment:
return ResponseEntity.ok("DevShop");

body's kunnen gewoon in de .ok( ) function meegegeven worden